chart/flux/README.md

```diff
@@ -211,8 +211,8 @@ The following tables lists the configurable parameters of the Flux chart and the
 | `git.url`                                         | `None`                                               | URL of git repo with Kubernetes manifests
 | `git.branch`                                      | `master`                                             | Branch of git repo to use for Kubernetes manifests
 | `git.path`                                        | `None`                                               | Path within git repo to locate Kubernetes manifests (relative path)
-| `git.user`                                        | `Weave Flux`                                         | Username to use as git committer
-| `git.email`                                       | `support@weave.works`                                | Email to use as git committer
+| `git.user`                                        | `Flux`                                               | Username to use as git committer
+| `git.email`                                       | `flux-dev@googlegroups.com`                          | Email to use as git committer
```

Comment:
This is a breaking change*, so I don't recommend including it in this PR.

*Why? Because if you installed the chart without supplying a value, installing the new chart will change the git committer.

Reply:
Similarly all the arguments etc below.

Reply:
Oh, right. So much for fixing things on the way :)

Reply:
Yeah I know, sorry about that!